src/main/webapp/WEB-INF/pages/checkout.jsp

```diff
@@ -0,0 +1,102 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
```

Comment:
overview.jsp and checkout.jsp files have duplicate code fragments. For the future: think of how to avoid this.